conf/carbon.properties.template

```diff
@@ -20,7 +20,7 @@
 #Mandatory. Carbon Store path
 carbon.storelocation=hdfs://hacluster/Opt/CarbonStore
 #Base directory for Data files
-carbon.ddl.base.hdfs.url=hdfs://hacluster/opt/data
+carbon.ddl.base.hdfs.url=
```

Comment:
If the default value is empty, shall we remove property from template or just comment out and keep ?

Reply:
I have commented it for now.